util.js

```diff
@@ -7,6 +7,18 @@ const transform = require('gl-vec3/transformMat4');
 
 const SVG_NS = 'http://www.w3.org/2000/svg';
 
+// Taken from https://github.com/yuzhe-han/ParentNode-replaceChildren
+// This is to support browsers that do not yet support `replaceChildren`
+const replaceChildrenPolyfill = function (...addNodes) {
+  while (this.lastChild) {
+    this.removeChild(this.lastChild);
+  }
+
+  if (addNodes !== undefined) {
```

Comment:
`addNodes` is never `undefined` because you used the rest operator. It's always going to be an array.

I guess this is what you meant to do?

```suggestion
  if (addNodes.length > 0) {
```